src/main/java/school/lemon/changerequest/java/introduction/hw2/Task1.java

```diff
@@ -1,7 +1,14 @@
 package school.lemon.changerequest.java.introduction.hw2;
 
+import java.io.PrintStream;
```

Comment:
please remove this import. Also before commit - check that you don't have any unused imports and your code is pretty-formatted.
In IDEA you can use following shortcuts:
`CTRL+ALT+L` - to format code
`CTRL+ALT+O` - to remove unused imports